elide-standalone/src/main/java/com/yahoo/elide/standalone/Util.java

```diff
@@ -52,13 +53,31 @@ public static EntityManagerFactory getEntityManagerFactory(String modelPackageNa
         }
 
         PersistenceUnitInfo persistenceUnitInfo = new PersistenceUnitInfoImpl("elide-stand-alone",
-                getAllEntities(modelPackageName), options);
+                combineModelEntities(modelPackageName, asyncModelPackageName), options);
 
         return new EntityManagerFactoryBuilderImpl(
                 new PersistenceUnitInfoDescriptor(persistenceUnitInfo), new HashMap<>())
                 .build();
     }
 
+    /**
+     * Combine the model entities with Async model.
+     *
+     * @param modelPackageName Package name
+     * @param asyncModelPackageName Async model package Name
+     * @return All entities combined from both package.
+     */
+    public static List<String> combineModelEntities(String modelPackageName, String asyncModelPackageName) {
+
+        List<String> modelEntities = getAllEntities(modelPackageName);
+
+        if (asyncModelPackageName != null) {
+            modelEntities.addAll(getAllEntities(asyncModelPackageName));
```

Comment:
asyncModelPackageName can just be AsyncQuery.class.getPackage().getName()

Reply:
Done